frontendbuild.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -ev
+
+if [ ! -f config.json ]; then
+  cp example-config.json config.json
+fi
```

Comment:
The grunt file consumes those settings and this should prevent it from erroring.